core/src/main/scala/kafka/admin/ConsumerGroupCommand.scala

```diff
@@ -87,7 +87,8 @@ object ConsumerGroupCommand extends Logging {
       }
     } catch {
       case e: Throwable =>
-        printError(s"Executing consumer group command failed due to ${e.getMessage}", Some(e))
+        printError(s"Executing consumer group command failed (${e.getMessage}) ${System.lineSeparator()}" +
+          s"${if (e.getCause != null) e.getCause}", Some(e))
```

Comment:
I'm not sure if we can depend on the cause giving us a useful `toString`. I wonder if we should just have `printError` print the full stack trace to stderr?

Reply:
Yeah, I can't think of a reason not to and it saves users the trouble of having to find the stack trace wherever the logging is putting it.